pkg/applicationserver/observability.go

```diff
@@ -101,6 +101,36 @@ var (
 		events.WithDataType(&ttnpb.ApplicationUp{}),
 		events.WithPropagateToParent(),
 	)
+	evtForwardDownlinkQueueInvalidated = events.Define(
+		"as.up.downlink.queue.invalidated.forward", "forward downlink queue invalidated",
```

Comment:
I know it's confusing, but this should fall under the `as.down.data` hierarchy, like below.

Reply:
Fixed!